plugins/rias/ls.rias.js

```diff
@@ -83,6 +84,7 @@
 
 		setOption = function(attr, run){
 			var attrVal = elem.getAttribute('data-'+ attr);
+			attrVal = elemStyles.getPropertyValue('--' + attr).trim() || attrVal;
```

Comment:
That's a good reason to add a prefix:) Pushed some changes with that and have also made the data attribute override the CSS variable.